make/photon/registry/Dockerfile.binary

```diff
@@ -7,4 +7,7 @@ ENV GO111MODULE auto
 WORKDIR $DISTRIBUTION_DIR
 COPY . $DISTRIBUTION_DIR
 
-RUN CGO_ENABLED=0 make PREFIX=/go clean binaries
+RUN CGO_ENABLED=0 make clean bin/registry
+
+FROM scratch
+COPY --from=build /go/src/github.com/docker/distribution/bin/registry /
```

Comment:
to /bin/binary?

Reply:
It was originally created in `binary`, not `bin/binary`, does that mean it should be changed to `bin/binary`?